src/internal/operations/restore.go

```diff
@@ -430,7 +430,9 @@ func formatDetailsForRestoration(
 
 	if sel.Service == selectors.ServiceOneDrive ||
 		sel.Service == selectors.ServiceSharePoint ||
-		sel.Service == selectors.ServiceGroups {
+		(sel.Service == selectors.ServiceGroups &&
+			len(paths) > 0 &&
+			(fds.Items()[0].Groups.ItemType == details.SharePointLibrary)) {
```

Comment:
I don't think this is the right check. I think we should probably have this check inside `AugmentRestorePath` on individual path entries.

Reply:
Yes. Doing the changes